src/BaGetter.Core/Configuration/StatisticsOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace BaGetter.Core;
+
+public class StatisticsOptions : IValidatableObject
+{
+    [Required]
+    public bool? HideServices { get; set; } = false;
```

Comment:
I'd prefer this to just be an additional key `HideStatsPage` in the outer `BaGetterOptions` scope, and if possible optional with a default of false.

Reply:
At the moment I would prefer that too.
I tried to think of other settings for the statistics, but nothing came up. So an extra section seems not justified.

Reply:
Sorry I haven't made this clear/obious. This setting (`HideServices`) only hides the listing of used services on the stats page. But after your comments we could add a setting to disable/hide the stats page completely (even from the page menu).

Reply:
So if I understand you correct, you want something like this:
```csharp
public class StatisticsOptions : IValidatableObject
{
    [Required]
    public bool? ShowStatisticsPage { get; set; } = true;

    [Required]
    public bool? HideUsedServices { get; set; } = false;
}
```

Reply:
Yes, option 1 disables the stats completely and the second option only hide the used services (maybe if you host a public BaGetter).

Reply:
Ah I see, without services the page still displays version and package stats. Ok that actually makes sense, I'm ok with the extra options then. Maybe name both in the same direction though, either both `Hide` or both `Show`. Or completely different `EnableStatisticsPage` and `IncludeConfiguredServices` (or `DisableStatisticsPage`/`ExcludeConfiguredServices`). I think I like `DisableStatisticsPage`/`ExcludeConfiguredServices` best but it's not a very strong preference. Probably because then the default can be false or null/not included in configuration.